CHANGELOG.next.md

```diff
@@ -17,6 +17,7 @@ Thanks, you're awesome :-) -->
 #### Added
 
 #### Improvements
+* description for `host.name` definition updated to encourage use of FDQN #2122
```

Comment:
Can we move this entry under the `8.7` header now that we're targeting it for `8.7`?